pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/plugin/PythonContainerPlugin.java

```diff
@@ -69,9 +76,7 @@ public void applyTo(final Project project) {
              * generic Python builds.  E.g. we make the pex task depend on it.
              */
             Task assemble = tasks.create(ApplicationContainer.TASK_ASSEMBLE_CONTAINERS);
-
-            // Add this no-op task for backward compatibility.  See PexExtension for details.
-            Task noop = tasks.create(PexExtension.TASK_BUILD_NOOP_PEX, NoopBuildPexTask.class);
+            // backward compatibility
```

Comment:
You can remove this comment; it doesn't add much here. (I prefer full sentences for comments anyway).

Reply:
Done.